<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCourse</class>
 <widget class="QDialog" name="EditCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <widget class="QLineEdit" name="txt_name"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Descrição</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="txt_description"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Preço</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="txt_price"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="edit_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>237</y>
     <width>269</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>EDITAR CURSO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_client_btn">
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>349</y>
     <width>269</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCELAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px</string>
   </property>
   <property name="text">
    <string>EDITAR CURSO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CANCEL">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>269</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCELAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
